docs/source/support/faq.rst

```diff
@@ -11,44 +11,33 @@ The answers in the FAQ regularly reference the network characters "**Alice**," "
 
 
-
-
 General
 -------
 
-.. _faq-testnet:
-
-Q: What is the network name for NuCypher's testnet?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-NuCypher currently has two testnets available, each of them with different goals:
-
-* ``lynx``: Public Long-Term Support testnet, intended as a stable playground for network users (e.g., Alices wishing to create sharing policies). Running on Ethereum Goerli testnet.
-* ``ibex``: Public testnet, intended as a playground for node operators (e.g., learning how to set up a PRE node etc.), as well as for internal development purposes. Running on Ethereun Rinkeby testnet.
-
-
-Using the PRE Application on Threshold Network
-----------------------------------------------
-
-*These are questions related to how the PRE Application on Threshold Network works.*
-
 Q: How do I know if the PRE Application is a good fit for my use case?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 A centralized approach to access control means users are forced to trust the system(s) with the plaintext version of
 their private data, and usually are given little insight into who exactly they are trusting. The PRE Application’s decentralized
 access control system offers developers, and their users, a departure from this opaque and trust-dependent paradigm.
-When integrated, `nucypher` enables end-to-end encrypted data sharing workflows within applications – but
+When integrated, ``nucypher`` enables end-to-end encrypted data sharing workflows within applications – but
 crucially, without sacrificing scalability, redundancy or performance – and applicable to data payloads of
 any form, size, structure, sensitivity or production cadence. Users enjoy the same sharing powers they
-currently take for granted, but are **not** obliged to trust the developers of the application or
+currently take for granted but are **not** obliged to trust the developers of the application or
 third-party access control services (e.g. centralized servers or key management systems) with their data.
 
 Q: How is PRE different from more traditional public key infrastructure (PKI)?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 For an in-depth analysis of this topic, see our blog post entitled "`Three things that NuCypher's "Ursula" can do that vanilla public-key cryptography can't <https://blog.nucypher.com/why-use-nucyphers-ursula-instead-of-traditional-public-key-cryptography/>`_".
 
+Q: If a dApp utilizes the PRE Application, is there an ETH gas fee associated with each re-encryption of encrypted data?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is a gas fee associated with granting and revoking policies. However, there is no gas fee per access.
+Previously policy management was done on L1 but the gas fees were high. Instead, policy management will now
```

Comment:
An attempt to remove the temporal language and state the facts:

Previously policies were published on ethereum mainnet.  As of nucypher 6.0 Policies are published to the polygon sidechain by default for lower gas fees.

Reply:
✔️